showkase-processor-testing/src/test/java/com/airbnb/android/showkase_processor_testing/BaseProcessorTest.kt

```diff
@@ -16,7 +16,7 @@ import java.io.File
  * Temporarily set this to true to have the test runner update test resource file expected outputs
  * instead of failing tests on mismatch. Use this to easily update expected outputs.
  */
-const val UPDATE_TEST_OUTPUTS = false
+const val UPDATE_TEST_OUTPUTS = true
```

Comment:
I'm not 100% sure about this but should we set this to false in the master branch?  Thinking since now, it will generate the test output always and not verify that the output is actually correct 🤔 
Wdyt? @vinaygaba :)

Reply:
Made https://github.com/airbnb/Showkase/pull/300 to address this :)